pkg/apis/faros/v1alpha1/gittrack_types.go

```diff
@@ -37,13 +47,17 @@ type GitTrackSpec struct {
 	DeployKey GitTrackDeployKey `json:"deployKey,omitempty"`
 }
 
-// GitTrackDeployKey holds a reference to an SSH key needed to access the repository
+// GitTrackDeployKey holds a reference to a secret such as an SSH key or OAuthToken needed to access the repository
```

Comment:
This still refers to `OAuthToken` but should be `HTTP Basic Auth` right?

Reply:
You're right. Fixing.